akka-actor/src/main/scala/akka/pattern/BackoffOptions.scala

```diff
@@ -555,6 +557,14 @@ trait BackoffOptions {
    */
   def withReplyWhileStopped(replyWhileStopped: Any): BackoffOptions
 
+  /**
+   * Predicate evaluated for each message, if it returns true and the supervised actor is
+   * stopped then the supervisor will stop its self. If it returns true while
+   * the supervised actor is running then it will be forwarded to the supervised actor and
+   * when the supervised actor stops its self the supervisor will stop its self.
+   */
+  def withFinalStopMessage(action: Any ⇒ Boolean): BackoffOptions
```

Comment:
`action` ? message?

Reply:
perhaps isFinalMessage

action was appropriate for the previous version that allowed you to do anything